clients/lsp-lua.el

```diff
@@ -127,6 +127,214 @@
   (and (f-exists? lsp-clients-lua-language-server-main-location)
        (f-exists? lsp-clients-lua-language-server-bin)))
 
+(defcustom lsp-lua-color-mode "Semantic" "Color mode."
+  :type '(choice (:tag "Grammar" "Semantic"))
+  :package-version '(lsp-mode . "7.0")
```

Comment:
please change these to 7.1. Can you also place the doc string on a new line, here and in the whole review?

Reply:
I've made the requested changes.
Thank you for the quick reply!